march_hardware/src/Joint.cpp

```diff
@@ -25,13 +25,13 @@ void Joint::prepareActuation()
                  this->name.c_str(), this->getActuationMode().toString().c_str());
   if (this->allowActuation)
   {
-    ROS_INFO("Preparing joint %s for actuation", this->name.c_str());
+    ROS_INFO("[%s] Preparing for actuation", this->name.c_str());
```

Comment:
Is this not a strange format? Now the message will look like 
"[LHAA] Preparing for actuation"

Reply:
it will look like:

    [left_hip_aa] Preparing for actuation